pkg/controllers/management/aks/aks_cluster_handler.go

```diff
@@ -390,30 +389,25 @@ func (e *aksOperatorController) deployAKSOperator() error {
 func (e *aksOperatorController) generateSATokenWithPublicAPI(cluster *mgmtv3.Cluster) (string, bool, error) {
 	restConfig, err := e.getKubeConfig(cluster)
 
-	netDialer := net.Dialer{
-		Timeout:   30 * time.Second,
-		KeepAlive: 30 * time.Second,
-	}
-	serviceToken, err := generateSAToken(restConfig, netDialer.DialContext)
+	// TODO: there will be a merge conflict here with another PR
```

Comment:
Which one? Is this still a concern?

Reply:
Yes. I just merged it, so I will fix them and update.